packaging/crio.conf.d/microshift-ovn.conf

```diff
@@ -2,8 +2,3 @@
 # ovn-kubernetes is the name configured by OVN-K in /etc/cni/net.d/ config file
 # by declaring this CRI-O will wait until that network is configured.
 cni_default_network = "ovn-kubernetes"
-
-[crio.runtime.workloads.management]
-activation_annotation = "target.workload.openshift.io/management"
```

Comment:
The annotations were also added to assets for several other components to make the cpu pinning effect, such as dns, service-ca, router and ovn. do we want to remove them or leave them in favor of future configurable changes?

Reply:
The annotations should remain since they won't be taken into effect until this config exists and like you said it'll give us the ability to easily decide on how we approach future configurability of this feature.

Reply:
make sense.